perspective.el

```diff
@@ -161,60 +173,19 @@ Run with the activated perspective active.")
 (define-key perspective-map (kbd "<left>") 'persp-prev)
 (define-key perspective-map persp-mode-prefix-key 'persp-switch-last)
 
+(defun persp--current-persp ()
```

Comment:
Please make this and `persp-last` public. We want users to have access to this information.

Reply:
Changed `persp--current-persp` and `persp--last-persp` to `persp-current-persp` and `persp-last-persp`.